addon/components/nrg-modal-container/modal-wrapper/component.js

```diff
@@ -28,8 +28,14 @@ export default Component.extend(ResizeMixin, {
   lightbox: alias('modal.lightbox'),
   modalClass: alias('modal.modalClass'),
   dimmerClass: alias('modal.dimmerClass'),
+  scrolling: alias('modal.scrolling'),
 
+  notMasterDetail: not('masterDetail'),
+  notLightbox: not('ligthbox'),
   notSidebar: not('sidebar'),
+  notRenderInPlace: not('renderInPlace'),
+  notSecondaryModalUse: and('notMasterDetail', 'notLightbox', 'notSidebar', 'notRenderInPlace'),
```

Comment:
This should be in the modal itself and not the wrapper..


```js
scrolling: and('notMasterDetail', 'notLightbox', 'notSidebar', 'notRenderInPlace'),
```

Reply:
Ahh I see what you were thinking now. Thanks!